hack/test-stirunimage.sh

```diff
@@ -71,69 +71,84 @@ trap cleanup EXIT SIGINT
 
-s2i build "${S2I_WORK_DIR}/cakephp-ex" openshift/php-55-centos7 test --loglevel=5 &> "${WORK_DIR}/s2i-abs-noproto.log"
-check_result $? "${WORK_DIR}/s2i-abs-noproto.log"
-
-## don't do ssh tests here because credentials are needed (even for the git user), which
-## don't exist in the vagrant/jenkins setup
-
-test_debug "s2i build with non-git repo file location"
-
-rm -rf "${WORK_DIR}/cakephp-ex/.git"
-s2i build "${S2I_WORK_DIR}/cakephp-ex" openshift/php-55-centos7 test --loglevel=5 --loglevel=5 &> "${WORK_DIR}/s2i-non-repo.log"
-check_result $? ""
-grep "Copying sources" "${WORK_DIR}/s2i-non-repo.log"
-check_result $? "${WORK_DIR}/s2i-non-repo.log"
-
-test_debug "s2i rebuild"
-s2i build https://github.com/openshift/sti-php.git --context-dir=5.5/test/test-app registry.access.redhat.com/openshift3/php-55-rhel7 rack-test-app --incremental=true --loglevel=5 &> "${WORK_DIR}/s2i-pre-rebuild.log"
-check_result $? "${WORK_DIR}/s2i-pre-rebuild.log"
-s2i rebuild rack-test-app:latest rack-test-app:v1 --force-pull=false -p never --loglevel=5 &> "${WORK_DIR}/s2i-rebuild.log"
-check_result $? "${WORK_DIR}/s2i-rebuild.log"
-
-test_debug "s2i usage"
-
-s2i usage openshift/ruby-20-centos7 --loglevel=5 &> "${WORK_DIR}/s2i-usage.log"
-check_result $? ""
-grep "Sample invocation" "${WORK_DIR}/s2i-usage.log"
-check_result $? "${WORK_DIR}/s2i-usage.log"
-
-test_debug "s2i build with git proto"
-
-s2i build https://github.com/openshift/cakephp-ex openshift/php-55-centos7 test --run=true --loglevel=5 &> "${WORK_DIR}/s2i-git-proto.log" &
-check_result $? "${WORK_DIR}/s2i-git-proto.log"
-
-test_debug "s2i build with --run==true option"
 if [[ "$OSTYPE" == "cygwin" ]]; then
-  ( cd hack/windows/sigintwrap && make )
-  hack/windows/sigintwrap/sigintwrap 's2i build https://github.com/bparees/openshift-jee-sample openshift/wildfly-90-centos7 test-jee-app --run=true --loglevel=5' &> "${WORK_DIR}/s2i-run.log" &
+   test_debug "s2i build with windows volume options"
+   s2i build https://github.com/openshift/cakephp-ex openshift/php-55-centos7 test --volume 'c:\:/home/:z' &> "${WORK_DIR}/s2i-volume-correct.log"
+   check_result $? "${WORK_DIR}/s2i-volume-correct.log"
+   ! s2i build https://github.com/openshift/cakephp-ex openshift/php-55-centos7 test --volume 'c:\:/home/:z:z' &> "${WORK_DIR}/s2i-volume-incorrect.log"
+   check_result $? "${WORK_DIR}/s2i-volume-incorrect.log"
 else
-  s2i build https://github.com/bparees/openshift-jee-sample openshift/wildfly-90-centos7 test-jee-app --run=true --loglevel=5 &> "${WORK_DIR}/s2i-run.log" &
+   test_debug "s2i build with unix volume options"
+   s2i build https://github.com/openshift/cakephp-ex openshift/php-55-centos7 test --volume "${S2I_WORK_DIR}:/home/:z" &> "${WORK_DIR}/s2i-volume-correct.log"
+   check_result $? "${WORK_DIR}/s2i-volume-correct.log"
+   ! s2i build https://github.com/openshift/cakephp-ex openshift/php-55-centos7 test --volume "${S2I_WORK_DIR}:/home/:z:z" &> "${WORK_DIR}/s2i-volume-incorrect.log"
```

Comment:
given that you're getting one "TEST PASSED" after the "s2i build with unix volume options", it would seem like the test is hanging on this second test.  have you tried running this command by hand?